kview/src/main/java/dev/ikm/komet/kview/mvvm/view/properties/PropertiesController.java

```diff
@@ -329,6 +357,32 @@ public void initialize() throws IOException {
         };
         eventBus.subscribe(conceptTopic, OpenPropertiesPanelEvent.class, propsPanelOpen);
 
+        addStampSubscriber = evt -> {
+            contentBorderPane.setCenter(stampAddPane);
+            editButton.setSelected(true);
+
+            stampJFXNode.updateViewModel("stampViewModel", stampViewModel -> {
+                EntityVersion latestVersion = viewProperties.calculator().latest(entityFacade).get();
+                StampEntity stampEntity = latestVersion.stamp();
+
+                stampViewModel.setPropertyValue(ENTITY, entityFacade)
+                              .setPropertyValue(PREV_STAMP, stampEntity)
+                              .setPropertyValue(StampViewModel2.StampProperties.STATUS, stampEntity.state())
+                              .setPropertyValue(StampViewModel2.StampProperties.MODULE, stampEntity.module())
+                              .setPropertyValue(PATH, stampEntity.path());
+
+                stampViewModel.setPropertyValues(MODULES, fetchDescendentsOfConcept(viewProperties, TinkarTerm.MODULE.publicId()))
+                              .setPropertyValues(PATHS, fetchDescendentsOfConcept(viewProperties, TinkarTerm.PATH.publicId()))
+                              .setPropertyValues(STATUSES, List.of(State.values()));
```

Comment:
Can you provide the `true` as the last parameter of the `setPropertyValues(propertyname, value, skip)` method?
Skip to avoid `save()` and `reset()` to copy items from property layer to model value layer.